eq-author/src/apollo/sentryUtils.js

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from "@sentry/browser";
+import jwt from "jsonwebtoken";
+
+export const setSentryUser = token => {
+  const decodedToken = jwt.decode(token);
```

Comment:
If this gets an invalid token it returns `null`. This will cause the email and user_id lookups to blow up. We then lose the graphql error.

Reply:
Ive added code and tests to stop this happening now